pkg/ingester/transfer.go

```diff
@@ -29,18 +34,112 @@ var (
 		Name: "cortex_ingester_received_chunks",
 		Help: "The total number of chunks received by this ingester whilst joining",
 	})
+	sentFiles = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "cortex_ingester_sent_files",
```

Comment:
Would `cortex_ingester_transfer_out_files` and `cortex_ingester_transfer_in_files` more clear?